addons/stock/tests/__init__.py

```diff
@@ -4,3 +4,4 @@
 import test_product
 import test_warehouse
 import test_shipment
+from . import test_stock_location_search
```

Comment:
@moylop260 I actually didn't notice this, thanks for pointing this out! Sorry for this. :)